lib/pfconfig/manager.pm

```diff
@@ -39,233 +38,304 @@ use List::MoreUtils qw(any firstval uniq);
 use Scalar::Util qw(refaddr reftype tainted blessed);
 use UNIVERSAL::require;
 use Data::Dumper;
-use pfconfig::backend::bdb;
+use pfconfig::backend::mysql;
```

Comment:
This is should be moved to a factory if we want to support multiple backends at once